docs/table-index.md

```diff
@@ -13,6 +13,7 @@ This table can be found at the following URLs depending on the choice of format:
 | Name | Type | Description | Example |
 | ---- | ---- | ----------- | ------- |
 | **key** | `string` | Unique string identifying the region | US_CA_06001 |
+| **place_ID** | `string` | A textual identifier that uniquely identifies a place in the Google Places database and on Google Maps ([details](https://developers.google.com/places/web-service/place-id))| ChIJd_Y0eVIvkIARuQyDN0F1LBA |
```

Comment:
Can you please fix the capitalization? I think the name should be `place_id`

Reply:
```suggestion
| **place_id** | `string` | A textual identifier that uniquely identifies a place in the Google Places database and on Google Maps ([details](https://developers.google.com/places/web-service/place-id))| ChIJd_Y0eVIvkIARuQyDN0F1LBA |
```

Reply:
Done.